app/src/main/java/com/example/kursakademiaandroida/core/database/AppDatabase.kt

```diff
@@ -1,8 +1,14 @@
 package com.example.kursakademiaandroida.core.database
 
+import androidx.room.Database
 import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import com.example.kursakademiaandroida.features.episodes.data.local.EpisodeDao
+import com.example.kursakademiaandroida.features.episodes.data.local.model.EpisodeCached
 
-
+@Database(entities = [EpisodeCached::class], version = 1)
```

Comment:
Możesz dodać do tablicy klasy `CharacterCached` oraz `LocationCached`, zdefiniować `characterDao` oraz `locationDao`, a na końcu dodać te dao to modułu koina.